src/lib/components/Swap/Input.tsx

```diff
@@ -1,7 +1,9 @@
 import { Trans } from '@lingui/macro'
 import { useUSDCValue } from 'hooks/useUSDCPrice'
+import { useAtomValue } from 'jotai/utils'
 import { useSwapAmount, useSwapCurrency, useSwapInfo } from 'lib/hooks/swap'
 import { usePrefetchCurrencyColor } from 'lib/hooks/useCurrencyColor'
+import { localeAtom } from 'lib/state/locale'
```

Comment:
This should be taken from lingui's provider if it can be, instead of a new atom. Does lingui give us a hook to get this?

Reply:
Didn't know this was a thing. cool!

https://lingui.js.org/ref/react.html#uselingui

   `const { i18n } = useLingui()`